plugins/filter_kubernetes_sumologic.rb

```diff
@@ -59,6 +87,10 @@ def filter(tag, time, record)
 
         annotations = kubernetes.fetch('annotations', {})
 
+        if annotations['sumologic.com/excludeFromSumo'] == 'true'
```

Comment:
excludeFromSumo is prob not really needed. just having exclude or ignore is prob better.